docs/guides/streams/node-readable-to-json.md

```diff
@@ -2,7 +2,7 @@
 name: Convert a Node.js Readable to JSON
 ---
 
-To convert a Node.js `Readable` stream to a JSON object in Bun, you can create a new [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response) object with the stream as the body, then use [`response.json()`](https://developer.mozilla.org/en-US/docs/Web/API/Response/json) to read the stream into a JSON object.
+To convert a Node.js `Readable` stream to a JSON object in Bun, you can create a new [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response) object with the stream as the body, then use [`Response.json()`](https://developer.mozilla.org/en-US/docs/Web/API/Response/json) to read the stream into a JSON object.
```

Comment:
This is not correct, we're referring to the instance method and not the constructor method

Reply:
oh yeah my bad, i got the `return Response.json()` mixed up with this